helm/cluster-api/templates/delete-aggregated-roles/job.yaml

```diff
@@ -48,5 +39,11 @@ spec:
             drop:
             - ALL
           readOnlyRootFilesystem: true
+        image: '{{ .Values.images.domain }}/giantswarm/kubectl:1.32.1'
```

Comment:
Can we call out this upgrade in the changelog?

Also, should we add the Renovate comment here to have it automatically updated?

Reply:
Sure, gonna add a hint!

Reply:
I'm not sure about Renovate. Do you have an example for how to point to an image registry?